src/pools/weightedPool/weightedPool.ts

```diff
@@ -331,20 +336,24 @@ export class WeightedPool implements PoolBase<WeightedPoolPairData> {
      * @returns EVM scale.
      */
     _calcTokensOutGivenExactBptIn(bptAmountIn: BigNumber): BigNumber[] {
-        // token balances are stored in human scale and must be EVM for maths
-        const balancesEvm = this.tokens
+        // balances and amounts must be normalized to 1e18 fixed point - e.g. 1USDC => 1e18 not 1e6
+        // takes price rate into account
```

Comment:
Just remove this to be 100% clear

Reply:
Ops - that one slipped 😬 - I'll remove over here 👍